docs/howto/set-config.rst

```diff
@@ -46,7 +46,18 @@ the key like this:
     [core]
     sql_alchemy_conn_cmd = bash_command_to_run
 
-The following config options support this ``_cmd`` version:
+You can also derive the connection string at run time by appending ``_secret`` to
+the key like this:
+
+.. code-block:: ini
+
+    [core]
+    sql_alchemy_conn_secret = sql_alchemy_conn
```

Comment:
Hmmmmm, I wonder if this will be confusing for people running in Kube, who might expect this to come from a Kube secret.

(I don't have a suggestion on how to deal with this.)

Reply:
for Kubernetes, the section would be kubernetes_secrets:

![image](https://user-images.githubusercontent.com/8811558/86855309-fcb9c980-c0b1-11ea-9e21-09a3a737196b.png)